config/twilio-notification-channel.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return [
+    'username' => env('TWILIO_USERNAME'), // optional when using auth token
+    'password' => env('TWILIO_PASSWORD'), // optional when using auth token
+    'auth_token' => env('TWILIO_AUTH_TOKEN'), // optional when using username and password
+    'account_sid' => env('TWILIO_ACCOUNT_SID'),
+
+    'from' => env('TWILIO_FROM'), // optional
+    'alphanumeric_sender' => null,
+
+    'ignored_error_codes' => [
```

Comment:
TODO documentation comment here

Reply:
Maybe we should also allow passing `*` to suppress all notifications? That way it's backward compatible with the previous version if config is overridden.

Reply:
Yes, I think it's a good idea.